test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -373,6 +373,7 @@ Function sys.datetimeoffset_hash(sys.datetimeoffset)
 Function sys.datetimeoffset_larger(sys.datetimeoffset,sys.datetimeoffset)
 Function sys.datetimeoffset_smaller(sys.datetimeoffset,sys.datetimeoffset)
 Function sys.datetimeoffset_sqlvariant(sys.datetimeoffset)
+Function sys.datetimeoffsetfromparts(text,text,text,text,text,text,text,text,text,text)
```

Comment:
It means that there is no test case for this function. Could you add a test case?